cf-suse-stacks.yml

```diff
@@ -63,23 +73,39 @@ jobs: ##########################################################################
     - get: version
       params: { pre: alpha }
     - get: receipt
+
   - do:
     - task: make-rootfs
       privileged: true
       file: src-ci/tasks/make-rootfs/task.yml
       input_mapping:
         src: stacks-git
+
   - put: artifact
     params:
       file: out/opensuse42-*.tar.gz
       acl: public-read
+
   - put: version
     params: { file: version/version }
+
   - put: receipt
     params:
       file: out/opensuse42-*.receipt.txt
       acl: public-read
+
   - put: receipt-diff
     params:
       file: out/opensuse42-*.receipt.diff
       acl: public-read
+
+  - put: docker-image
+    params:
+      import_file: out/opensuse42.tar.gz
+      skip_download: true
```

Comment:
Does `skip_download` have an effect here? The [docker resource docs](https://github.com/concourse/docker-image-resource#parameters-1) use this for the `in` step, but not the `out` step.